### ISSUE TITLE
Add support for Wikibase federation

### ISSUE THREAD

**Issue by Tpt** [Wikidata/Wikidata-Toolkit#267]
Wikidata-Toolkit should support entity id values pointing to external repositories e.g. structured data on commons.

**Comment by wetneb** [comment]
Related to #474: in the deserialization of MediaInfo entities, the entity values are currently created with an incorrect site IRI.

**Comment by Tanish-Gupta** [comment]
Hi, 
I was going the repository, is this issue still open? I can look into the issue once if it is still pending

Thank you

**Comment by wetneb** [comment]
Yes it is, but this requires some important (and potentially complex) design decisions, so it is probably not the best pick if you are looking for an easy issue to start with.

For this we are quite limited by design decisions on Wikibase's side, and these things are evolving quite rapidly at the moment, so I would not rush to implement this without coordinating seriously with WMDE.

**Comment by Tanish-Gupta** [comment]
Okay, thank you for your response.
